src/xunit.performance.api/Model/AssemblyModel.cs

```diff
@@ -175,6 +175,143 @@ private static IEnumerable<(string testCaseName, string metric, IEnumerable<doub
     }
 
     [Serializable]
+    [XmlRoot("BenchmarkScenario")]
+    public sealed class BenchmarkScenario
+    {
+        [XmlAttribute("Name")]
+        public string Name { get; set; }
+
+        [XmlAttribute("Namespace")]
+        public string Namespace { get; set; }
+        
+        [XmlArray("Assemblies")]
+        public List<NetcoreAssembly> Assemblies { get; set; }
```

Comment:
Do scenarios need an assembly list?  Since they have no coupling to xunit or any other assembly based runner, I wouldn't expect assemblies to be tracked here.